### ISSUE TITLE
DeathAdder Chroma not responding

### ISSUE THREAD

**Issue by voydz** [1kc/razer-macos#198]
Hey there,

this project is super awesome. Thanks for the hard work. I love it! So I am using this with my BlackWidow Chroma and DeathAdder Chroma (yes, older Hardware). This is the exact product lineup where Razer put out a massive sh***show in its software. The keyboard is only supported on Synapse 3 (not working on Mac) and the mouse only works with Synapse 2 (not working anymore for me on Mac).

I am able to configure both devices using Synapse 3 and 2 respectively on Windows.
This tool works fine for my keyboard but not for my mouse, tho. I checked the FAQ and found my issue best described in `Q: Selecting a colour setting has no effect on my keyboard.` so I checked the profile. I tried to reset everything but it does not work anyway.

**TLDR:** Now for the issue I have: are there known issues with the DeathAdder Chroma not responding to changes in dpi/color of this tool?

As I know a thing or too about electron and node and react stuff, I would love to help with this. So, if anyone can give me a pointer where I should start looking. That would be 👍 

best regards
